classes/Company.php

```diff
@@ -0,0 +1,224 @@
+<?php
+class Company {
+    private $conn;
+    private $table = 'companies';
+
+    public $id;
+    public $company_name;
+    public $email;
+    public $country;
+    public $state;
+    public $city;
+    public $specialization;
+    public $contact;
+    public $contact_person;
+
+    public $search_string;
+
+    public $created_at;
+    
+
+    public function __construct($db) {
+        $this->conn = $db;
+    }
+
+    public function create() {
+        try{
+            $query = 'INSERT INTO ' . $this->table . ' SET
+                company_name = :company_name,
+                email = :email,
+                country = :country,
+                state = :state,
+                city = :city,
+                specialization = :specialization,
+                contact = :contact,
+                contact_person = :contact_person';
+
+            $stmt = $this->conn->prepare($query);
+
+            $this->company_name = ucwords(htmlspecialchars(strip_tags($this->company_name)));
+            $this->email = ucwords(htmlspecialchars(strip_tags($this->email)));
+            $this->country = htmlspecialchars(strip_tags($this->country));
+            $this->state = ucwords(htmlspecialchars(strip_tags($this->state)));
+            $this->city = ucwords(htmlspecialchars(strip_tags($this->city)));
+            $this->specialization = ucwords(htmlspecialchars(strip_tags($this->specialization)));
+            $this->contact = htmlspecialchars(strip_tags($this->contact));
+            $this->contact_person = htmlspecialchars(strip_tags($this->contact_person));
+
+            $stmt->bindParam(':company_name', $this->company_name);
+            $stmt->bindParam(':email', $this->email);
+            $stmt->bindParam(':country', $this->country);
+            $stmt->bindParam(':state', $this->state);
+            $stmt->bindParam(':city', $this->city);
+            $stmt->bindParam(':specialization', $this->specialization);
+            $stmt->bindParam(':contact', $this->contact);
+            $stmt->bindParam(':contact_person', $this->contact_person);
+
+            if($stmt->execute()) {
+                return true;
+            }
+
+            return false;
+        }catch(PDOException $e){
+            $timestamp = date('Y-m-d H:i:s'); 
+            error_log("$timestamp: Something went wrong while creating a company in the database! \n", 3, 'error_log'); 
```

Comment:
Log the actual exception $e so we can see the issue in the log file for troubleshooting.